projects/piex/Dockerfile

```diff
@@ -16,6 +16,12 @@
 
 FROM gcr.io/oss-fuzz-base/base-builder
 RUN apt-get update && apt-get install -y make autoconf automake libtool
-RUN git clone https://github.com/guidovranken/piex.git piex
+RUN git clone --depth 1 https://android.googlesource.com/platform/external/piex
+ADD https://storage.googleapis.com/clusterfuzz-builds/piex/piex-address-202411210638.zip /tmp/piex.zip
```

Comment:
what's this needed for? can you add a comment?

Reply:
I'm assuming this is for the seed corpus only?

Reply:
Yep